panda-utilities/src/test/java/org/panda_lang/utilities/commons/iterable/ArrayIterableTest.java

```diff
@@ -0,0 +1,26 @@
+package org.panda_lang.utilities.commons.iterable;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Iterator;
+
+class ArrayIterableTest {
+
+    @Test
+    void testArrayIterable() {
+        ArrayIterable<String> arrayIterable =
+                new ArrayIterable<>(new String[]{"a", "b", "c"});
```

Comment:
We don't limit line length to 80 characters. The recommended line length is 120, the hard limit is 160.

Reply:
Please check the commit bee18e9 to make sure it's fixed.